```
       > 2025/03/21 13:29:43 ...migrations/common.go:18:WarnAndNotice() [W] PR #3 in <GitlabDownloader http://127.0.0.1:42967 [6590996]/archbuild> has invalid HeadRef: cgtx:review
       > --- FAIL: TestURLAllowedSSH (0.02s)
       >     --- FAIL: TestURLAllowedSSH/Pushmirror_URL (0.00s)
       >         migrate_test.go:128:
       >                Error Trace:    /build/source/services/migrations/migrate_test.go:128
       >                Error:          Received unexpected error:
       >                                migration/cloning from 'git.gay' is not allowed.
       >                Test:           TestURLAllowedSSH/Pushmirror_URL
       > FAIL
       > FAIL   code.gitea.io/gitea/services/migrations 0.758s
       > FAIL
```

looking into why this test fails

--- a/pkgs/by-name/fo/forgejo/generic.nix
+++ b/pkgs/by-name/fo/forgejo/generic.nix
@@ -119,12 +119,11 @@ buildGoModule rec {
   checkFlags =
     let
       skippedTests = [
-        "Test_SSHParsePublicKey/dsa-1024/SSHKeygen" # dsa-1024 is deprecated in openssh and requires opting-in at compile time
-        "Test_calcFingerprint/dsa-1024/SSHKeygen" # dsa-1024 is deprecated in openssh and requires opting-in at compile time
         "TestPassword" # requires network: api.pwnedpasswords.com
         "TestCaptcha" # requires network: hcaptcha.com
         "TestDNSUpdate" # requires network: release.forgejo.org
         "TestMigrateWhiteBlocklist" # requires network: gitlab.com (DNS)
+        "TestURLAllowedSSH/Pushmirror_URL" # requires network git.gay (DNS)
       ];
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];

--- a/pkgs/by-name/fo/forgejo/package.nix
+++ b/pkgs/by-name/fo/forgejo/package.nix
@@ -1,8 +1,8 @@
 import ./generic.nix {
-  version = "10.0.1";
-  hash = "sha256-JLy/edAkbsOqQTQqqoyxc0UX9fPnP8HD54IUzcsyGe4=";
+  version = "10.0.2";
+  hash = "sha256-UxjhJA6o1LS/7/s0BzcaTSkXSsnKfjIMIjL6ASSot6k=";
   npmDepsHash = "sha256-e3SE6cu1xCBdoMRqp2Gcjcay/EwjF+bTdPOlpL1STvw=";
-  vendorHash = "sha256-fqHwqpIetX2jTAWAonRWqF1tArL7Ik/XXURY51jGOn0=";
+  vendorHash = "sha256-ceV73QmxYEthjsA50ylojwC4dcTYAbE2UgxuxFqvi24=";
   lts = false;
   nixUpdateExtraArgs = [
     "--override-filename"

